tests/providers/microsoft/azure/operators/test_azure_service_queue.py

```diff
@@ -0,0 +1,234 @@
+        self.dag = DAG(DAG_ID, default_args=args)
+
+    def test_init(self):
+        """
+        Test init by creating AzureServiceBusSendMessageOperator with task id, queue_name, message,
+        batch and asserting with values
+        """
+        msg = "test message"
+        asb_send_message_queue_operator = AzureServiceBusSendMessageOperator(
+            task_id="asb_send_message_queue_without_batch",
+            queue_name=QUEUE_NAME,
+            message=msg,
+            batch=False,
+            dag=self.dag,
+        )
+        assert asb_send_message_queue_operator.task_id == "asb_send_message_queue_without_batch"
+        assert asb_send_message_queue_operator.queue_name == QUEUE_NAME
+        assert asb_send_message_queue_operator.message == msg
+        assert asb_send_message_queue_operator.batch is False
+
+        asb_send_message_queue_operator = AzureServiceBusSendMessageOperator(
+            task_id="asb_send_message_queue_with_batch",
+            queue_name=QUEUE_NAME,
+            message=msg,
+            batch=True,
+            dag=self.dag,
+        )
+        assert asb_send_message_queue_operator.task_id == "asb_send_message_queue_with_batch"
+        assert asb_send_message_queue_operator.queue_name == QUEUE_NAME
+        assert asb_send_message_queue_operator.message == msg
+        assert asb_send_message_queue_operator.batch is True
+
+    @mock.patch("airflow.providers.microsoft.azure.hooks.asb_message.ServiceBusMessageHook.get_conn")
+    def test_send_message_queue(self, mock_get_conn):
+        """
+        Test AzureServiceBusSendMessageOperator with queue name, batch boolean flag, mock
+        the send_messages of azure service bus function
+        """
+        asb_send_message_queue_operator = AzureServiceBusSendMessageOperator(
+            task_id="asb_send_message_queue",
+            queue_name=QUEUE_NAME,
+            message="Test message",
+            batch=False,
+            dag=self.dag,
+        )
+        asb_send_message_queue_operator.execute(None)
+        expected_calls = [
+            mock.call()
+            .__enter__()
+            .get_queue_sender(QUEUE_NAME)
+            .__enter__()
+            .send_messages(ServiceBusMessage("Test message"))
+            .__exit__()
+        ]
+        mock_get_conn.assert_has_calls(expected_calls, any_order=False)
+
+
+class TestAzureServiceBusReceiveMessageOperator(unittest.TestCase):
+    def setUp(self):
+        args = {'owner': OWNER_NAME, 'start_date': datetime.datetime(2017, 1, 1)}
+        self.dag = DAG(DAG_ID, default_args=args)
+
+    def test_init(self):
+        """
+        Test init by creating AzureServiceBusReceiveMessageOperator with task id, queue_name, message,
+        batch and asserting with values
+        """
+
+        asb_receive_queue_operator = AzureServiceBusReceiveMessageOperator(
+            task_id="asb_receive_message_queue",
+            queue_name=QUEUE_NAME,
+            dag=self.dag,
```

Comment:
nit: How about removing `dag` param since the intention here is to test the task only if you do so you do not need `setUp` method also

Reply:
Removed `dag=self.dag`